### DISCUSSION TITLE
android platformsdk parameter doesn't work

### DISCUSSION THREAD

**Discussion by Nisaba** [BowlerHatLLC/vscode-as3mxml#688] [Q&A] [answered]
Hello,

Now we must specify the Android SDK path to compile Android App to the latest versions...

But I can't specify it on the asconfig.json.
Here is what I have : 

```
			 "android": {
				"output": "bin-release/GeoTracker.apk",
				"platformsdk": "f:\\ANDROID\\ANDROIDSDK",
				"listen": true,
				"signingOptions": {
					...
				 }
			},

```

But I get this error:
platformsdk must refer to a valid directory path - F:\CODES-SOURCES\MyProject\ANDROIDSDK

So I have to copy the SDK inside the project folder. It works but it's very inefficient...

Thank you for your help

**Comment by joshtynjala** [comment]
Does it work if you switch from backslashes to forward slashes?

This error comes from the adt tool in the AIR SDK, so there's not much I can do to help on the vscode-as3mxml side of things.

**Comment by Nisaba** [comment]
Thank you for your quick reply.

I've the same issue with forward slashes:
platformsdk must refer to a valid directory path - F:\CODES-SOURCES\GeoLoc-AppStore\GeoLoc-Offline\ANDROIDSDK

Strange, because the backslashes work well in settings.json:

```
{
    "as3mxml.sdk.framework": "f:\\CODES-SOURCES\\SDKs\\Flex-Sdk-4.16.1-Air50-Harman",
    "git.ignoreLimitWarning": true
}


```

**Reply by joshtynjala** [reply]
> I've the same issue with forward slashes

Then perhaps the issue isn't with backslashes. I mentioned it because I saw mention of forward slashes on Windows here:

https://github.com/airsdk/Adobe-Runtime-Support/issues/1506#issuecomment-1002608223

**Reply by Nisaba** [reply, marked as answer]
Great ! It works now.
Your comment has suggest to me to have a look in adt.cfg
The problem was about an old and bad value in adt.cfg, located not in the lib folder of the Air SDK but in my homepath folder...

Thank you !
